teams/index.html

```diff
@@ -151,6 +151,23 @@ <h4 class="text-center">Kenny Gillen</h4>
                       </button>
                     </div>
                 </div>
+                <div class="column">
+                    <a class="team-member" data-open="modal21">
+                        <img class="thumbnail" alt="photo of Riad Gozim" src="{{ site.baseurl }}/img/team/riad_gozim.jpg">
+                        <h5>Riad Gozim</h5>
+                        <p class="job-title">Software Developer</p>
+                        <span class="bio-link tiny button hollow">Read Bio</span>
+                    </a>                                                            
+                    <div class="reveal" id="modal21" data-reveal>
+                      <h4 class="text-center">Riad Gozim</h4>
+                      <p class="job-title">Software Developer</p>
+                      <hr>
+                      <p>Riad Gozim is a Software Developer who joined the OME team in July 2017. He studied Computer Games Technology in Dundee. Since graduating, he worked in the start-up scene where he helped grow one of Scotland's most successful mobile development studios. He has built mobile and server software for various corporations in different industries, including the banking and the pharmaceutical sector and, in 2014, he developed software for the world's first mobile Parkinson's study. In his spare time he enjoys weight lifting, hill walking and cooking, when he can find the ingredients.</p>
```

Comment:
Can we wrap this to 80 columns?  It's very difficult to read!

Reply:
It's staged on snoopy as linked above but yes, I'll reformat this page